SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/Microsoft.AppCenter.Data.Shared.projitems

```diff
@@ -21,5 +21,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)DocumentMetadata.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)DefaultPartitions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)RemoteOperationCompletedEventArgs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ConversionExtensions.cs" />
```

Comment:
Please rename the file as the content is a different class than the file name, it's confusing.